templates/pages/page.html.twig

```diff
@@ -398,14 +398,11 @@
 
                                             </div>
 
-                                            {#
-                                            <div class="govuk-grid-column-one-third apollo-enclosure">
-                                                <h3 class="govuk-heading-s">Talk to us</h3>
-                                                <p>CCS customer services team is available Monday to Friday, 9am to 5pm.</p>
-
-                                                <h3 class="govuk-heading-s" style="margin-bottom: 0;">Talk to us</h3>
-                                                <p>0345 410 2222</p>
-                                            </div>#}
+                                             <div class="govuk-grid-column-one-third" ;  style=" padding:18px; background:#f8f8f8;:font-family: Helvetica, Arial, sans-serif; font-size: 12px; line-height: 23px" >
```

Comment:
Inline styles should only be used in very very few circumstances.

It's generally much better to put these styles in a stylesheet and then attach them using a class selector.

So remove 

`style="..."`

And put this is in a stylesheet. You may need to add a new class to this div so that you can select it as appropriate in CSS.

Although in this specific instance, I think you should be able to remove the custom style here, and just add the `wysiwyg-content` class to this div element.

This class is sometimes necessary to ensure that text displays properly.